x-pack/plugins/fleet/server/integration_tests/ha_setup.test.ts

```diff
@@ -10,7 +10,12 @@ import Path from 'path';
 import { range } from 'lodash';
 
 import type { ISavedObjectsRepository } from '@kbn/core/server';
-import * as kbnTestServer from '@kbn/core/test_helpers/kbn_server';
+import type { TestElasticsearchUtils, createRoot } from '@kbn/core-test-helpers-kbn-server';
```

Comment:
@elastic/fleet I split the imports out into separate methods and/or types after migraitng the `kbn_server` utilities to a package.